docs/lara-plugin-api/interfaces/isidebarcontroller.md

```diff
@@ -23,7 +23,7 @@
 
 **● close**: *`function`*
 
-*Defined in [api/sidebar.ts:27](https://github.com/concord-consortium/lara/blob/3aa9451f/lara-plugin-api/src/api/sidebar.ts#L27)*
+*Defined in [api/sidebar.ts:27](https://github.com/concord-consortium/lara/blob/4825314e/lara-plugin-api/src/api/sidebar.ts#L27)*
```

Comment:
Can the documentation generator be configured to render these urls as:
`(../../../lara-plugin-api/src/api/sidbar.tx#L27)`?

I made a test branch to verify that this works:
https://github.com/concord-consortium/lara/blob/relative-md-test/docs/lara-plugin-api/interfaces/iclassinfo.md

Because the of the `../../..` requirement it would require some complex support from the documentation generator.

It'd be nice so documentation rebuilds don't result in changes for all of the generated files.